upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -618,10 +617,13 @@ func awsBuildCloudInstanceGroup(c AWSCloud, ig *kops.InstanceGroup, g *autoscali
 			glog.Warningf("ignoring instance with no instance id: %s in autoscaling group: %s", id, cg.HumanName)
 			continue
 		}
-		currentConfigName, err := findInstanceLaunchConfiguration(i)
-		if err != nil {
-			return nil, err
+		// @step: check if the instance is terminating
```

Comment:
Perhaps I should just drop this check? .. its wasn't here before .. @KashifSaadat .. what you reckon?

Reply:
No I think it's a sensible check to have, as the terminating instance shouldn't be included in the group generated below. Maybe just validate as part of the rolling-update / validation behaviour that this hasn't regressed.